app/src/main/res/layout-sw600dp-port/admin_pin_activity.xml

```diff
@@ -23,13 +23,16 @@
       android:paddingStart="104dp"
       android:paddingEnd="104dp"
       android:background="@color/oppiaBackgroundYellowIvory"
-      android:paddingBottom="96dp">
+      android:paddingBottom="96dp"
+      android:clipToPadding="false">
 
       <View
-        android:layout_width="0dp"
+        android:layout_width="match_parent"
         android:layout_height="6dp"
         android:background="@drawable/toolbar_drop_shadow"
         app:layout_constraintEnd_toEndOf="parent"
+        android:layout_marginLeft="-104dp"
```

Comment:
is this causing the failure?

Reply:
No, i've a solution

The error is in launching the AddProfileAcitivity it expects a sw600dp file to navigate so I'll add the XML for android in the sw600dp and after merging it I'll solve fix the other pr

Reply:
@anandwana001 i've used negative margin in my 2nd commit which passed all checks so that's not the cause for failure

Reply:
@FareesHussain Please do not mention negative margins/padding in code. Thats not a good practice.